### PR TITLE
Added more data types in dynamodb integration test

### PR DESCRIPTION
**Pull request by Deep1998**
Added more data types in dynamodb integration test

### CODE REVIEW

File: testing/dynamodb/integration_test.go [hunk 1]
```diff
@@ -91,6 +113,12 @@ func initIntegrationTests() (cleanup func()) {
 		log.Fatalf("cannot create databaseAdmin client: %v", err)
 	}
 
+	cfg := aws.Config{
+		Endpoint: aws.String("http://localhost:8000"),
```
Comment (hengfengli) [inline testing/dynamodb/integration_test.go:117]: Does this integration test run against a local emulator?

File: testing/dynamodb/integration_test.go [hunk 2]
```diff
@@ -228,17 +265,24 @@ func checkRow(ctx context.Context, t *testing.T, client *spanner.Client) {
 			t.Fatal(err)
 			break
 		}
-		var year, title, plot string
-		if err := row.Columns(&year, &title, &plot); err != nil {
+		var AttrInt, AttrFloat big.Rat
```
Comment (hengfengli) [inline testing/dynamodb/integration_test.go:268]: Why AttrInt and AttrFloat are big.Rat instead of int64 and float64?

Comment (hengfengli) [reply testing/dynamodb/integration_test.go:268]: I forgot this. The `Number` type in dynamodb will be converted to `Numeric` or `String` in spanner.

**Review by hengfengli — APPROVED** [review]
LGTM.

File: testing/dynamodb/integration_test.go (hunk 1 above)
Comment (Deep1998) [reply testing/dynamodb/integration_test.go:117]: Correct it uses the amazon/dynamodb-local image.

**Review by bharadwaj-aditya — APPROVED** [review]
LGTM

**Pull request merged**